### ISSUE TITLE
Is it absolutely necessary to use GPU

### ISSUE THREAD

**Issue by tokyokuma** [code-iai/iai_kinect2#471]
It's a very simple question.
Is it absolutely necessary to use GPU for using iai_kinect2?
I have Jetson TX1, but I am doing development(ex:writing code)  with a PC with only CPU.

**Comment by madsherlock** [comment]
No, you can use a CPU. Both GPU and CPU can be used as the back-end for OpenCL, and this is utilized here.
